gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/wixToolset.kt

```diff
@@ -39,12 +39,18 @@ internal fun JvmApplicationContext.configureWix() {
     val fileName = "wix311"
     val zipFile = wixDir.resolve("$fileName.zip")
     val unzipDir = root.layout.buildDirectory.dir(fileName)
-    val download = root.tasks.maybeCreate(DOWNLOAD_WIX_TOOLSET_TASK_NAME, Download::class.java).apply {
+    val download = root.tasks.findByName(DOWNLOAD_WIX_TOOLSET_TASK_NAME) ?: root.tasks.maybeCreate(
```

Comment:
The `maybeCreate` suppose to return the task if it exists ([the source code](https://github.com/gradle/gradle/blob/8ec670c5eeddcdaba6ee840360cb4eeda788558e/subprojects/core/src/main/java/org/gradle/api/internal/AbstractNamedDomainObjectContainer.java#L60) calls `findByName`).

Does it definitely fixes the issue?

Reply:
yes, it fixes the issue.
The issue is casued by our `apply` block, which configures the task multiple times in our case, and each configuration adds a pair of `source` and `dest`. And apparently `dest` is supposed to be unique in Download task.